src/Fixer/ControlStructure/ControlStructureContinuationFixer.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ControlStructure;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class ControlStructureContinuationFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
+{
+    /**
+     * @internal
+     */
+    public const NEXT_LINE = 'next_line';
+
+    /**
+     * @internal
+     */
+    public const SAME_LINE = 'same_line';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Control structure continuation keyword must be on the configured line.',
+            [
+                new CodeSample(
+                    '<?php
+if ($baz == true) {
+    echo "foo";
+}
+else {
+    echo "bar";
+}
+'
+                ),
+                new CodeSample(
+                    '<?php
+if ($baz == true) {
+    echo "foo";
+} else {
+    echo "bar";
+}
+',
+                    ['keyword_position' => self::NEXT_LINE]
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return true;
```

Comment:
Can we have here `getControlContinuationTokens` call (converted to constant) and `T_DO`?

Reply:
Why `T_DO`?

Reply:
I added `T_WHILE`, which allowed me to remove the `fixDoWhile()`.